src/common/components/paymentMethods/creditCardForm/creditCardForm.tpl.html

```diff
@@ -71,7 +71,7 @@ <h4 class="panel-title  border-bottom-small  visible" translate>Credit Card Paym
       <div class="col-sm-4" ng-hide="$ctrl.paymentMethod">
         <div class="form-group is-required" ng-class="{'has-error': ($ctrl.creditCardPaymentForm.securityCode | showErrors)}">
           <label translate>CVV Code</label>
-          <input type="text" name="securityCode" class="form-control  form-control-subtle" ng-model="$ctrl.creditCardPayment.securityCode"ng-attr-placeholder="{{$ctrl.paymentMethod ? '****' : ''}}" ng-required="!$ctrl.paymentMethod || $ctrl.creditCardPayment.cardNumber">
+          <input type="text" name="securityCode" class="form-control form-control-subtle" ng-model="$ctrl.creditCardPayment.securityCode" ng-attr-placeholder="{{$ctrl.paymentMethod ? '****' : ''}}" ng-required="!$ctrl.paymentMethod && $ctrl.creditCardPayment.cardNumber">
```

Comment:
I think you can change `ng-required="!$ctrl.paymentMethod && $ctrl.creditCardPayment.cardNumber"` to `required`. This will make it always required unless the `ng-if` that I suggested you add above hides it because we are editing a payment.